fixtures/babel-component/.babelrc

```diff
@@ -3,5 +3,5 @@
     ["@babel/preset-env", { "targets": { "browsers": "last 1 version" } }],
     ["@babel/preset-react", { "runtime": "automatic" }]
   ],
-  "plugins": [["@compiled/babel-plugin", { "importReact": false }]]
+  "plugins": [["@compiled/babel-plugin", { "importReact": false, "optimizeCss": false }]]
```

Comment:
Needed to test the Webpack example -` yarn start:webpack`
When set to false, units won't be optimised. Eg `32px` won't be changed to `2pc`

![Screen Shot 2022-08-05 at 4 27 44 pm](https://user-images.githubusercontent.com/14813563/183015313-37ead849-b523-4add-9010-9cfcf7e55248.png)